src/extension/features/accounts/toggle-account-columns/index.jsx

```diff
@@ -19,19 +19,26 @@ export const ShowMemoButton = ({ defaultIsShown, id, toggleState }) => {
     <li>
       <div className="modal-account-view-menu" id={id}>
         <button
+          className={`ynab-checkbox ynab-checkbox-button modal-account-view-options-show-flag ${
+            defaultIsShown ? 'is-checked' : ''
+          }`}
           onClick={toggleHidden}
-          aria-label={l10n('toolkit.checkShowMenuColumn', 'Check Show Memo Column')}
           role="checkbox"
+          type="button"
```

Comment:
suggestion: We add a `aria-checked` attribute for accessibiltiy based on the value of `defaultIsShown`.